libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/NavigationOptionsEx.kt

```diff
@@ -0,0 +1,9 @@
+@file:JvmName("NavigationOptionsExtensions")
+
+package com.mapbox.navigation.base.internal.extensions
+
+import com.mapbox.navigation.base.internal.CopilotOptions
+import com.mapbox.navigation.base.options.NavigationOptions
+
+fun NavigationOptions.copilotOptions(copilotOptions: CopilotOptions) =
```

Comment:
Why not make it an extension for `NavigationOptions.Builder`? Do you think that invoking this extension on the `NavigationOptions` themselves is a more frequent use case?